### ISSUE TITLE
no permissions

### ISSUE THREAD

**Issue by johndoe71rus** [RingOV/gis-weather#31]
latest 0.8.2.61 git build
then run - no permissions

**Comment by RingOV** [comment]
This string must make gis-weather executable, but sometimes this is not worked.
https://github.com/RingOV/gis-weather/blob/master/scripts/build_deb.py#L66

If after build no permissions
`sudo chmod 755 /usr/bin/gis-weather`

**Comment by johndoe71rus** [comment]
yes, i find message about it at ubuntu-ru forum. but it was long ago. I think you know reason why this happened again
